bookkeeper-http/http-server/src/main/java/org/apache/bookkeeper/http/service/ServiceRequest.java

```diff
@@ -0,0 +1,71 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.apache.bookkeeper.http.service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.bookkeeper.http.HttpServer;
+
+/**
+ * A wrapper class that wrap a http request into a class which
+ * can then be passed into the service.
+ */
+public class ServiceRequest {
+    private String body;
+    private HttpServer.Method method = HttpServer.Method.GET;
+    private Map params = new HashMap();
```

Comment:
it is better to declare explicit types of the param, maybe <String,String> o <String, String[]>